ods_ci/tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -288,8 +288,8 @@ Notebook Expected Ide
 Wait Notebook To Be Loaded
     [Documentation]  Waits for the notebook IDE environment to be loaded completely and performs
     ...              a simple check with the Menu action in the loaded IDE.
-    [Arguments]  ${image_name}  ${version}
-    ${ide}=  Notebook Expected Ide  ${image_name}
+    [Arguments]  ${image}  ${version}
+    ${ide}=  Notebook Expected Ide  ${image}
```

Comment:
you're just changing variable name, what is this PR fixing?

Reply:
Few lines below, there was just `image` used in the method, which didn't work with the current state. My first intention was to rename that variable to `image_name` also, but then I realised that everywhere else we use just `image`...

Reply:
Ah, it's hidden bcos unchanged. Ok thanks